third_party/move/move-compiler-v2/src/pipeline/livevar_analysis_processor.rs

```diff
@@ -246,7 +246,14 @@ impl<'a> CopyTransformation<'a> {
         match bc {
             Assign(id, dst, src, kind) => match kind {
                 AssignKind::Inferred => {
-                    if self.check_implicit_copy(alive, id, false, src) {
+                    // TODO(https://github.com/aptos-labs/aptos-core/issues/11223):
```

Comment:
Sorry, since #11213 mentioned only `&mut` I thought it wasn't relevant for non-mut `&`, which is what has problems here.  We'll see if getting that in simplifies anything, but this issue is different.

Reply:
You could use TODO(#11223) here instead putting the whole url here.